### ISSUE TITLE
Unnecessary output device 

### ISSUE THREAD

**Issue by ilg-ul** [ysalathe/GenericUSBMIDI#5]
Hello Yves,

Can you check if the PC-300 has any MIDI input capabilities via USB?

I noticed that in the Finale NotePad MIDI setup, the PC-300 is also offered as choice for MIDI output, which might not be correct.

Regards,

Liviu

**Comment by elosha** [comment]
Yes, according to the manual, the PC-300 needs to be switched to "PC" mode, then it acts as MIDI interface when connected via USB.

Roland/Edirol PCR-80, -50 and -30 also have been advertised to have this capability, they even offer a MIDI input port for that cause. But I don't know if it needs a driver or is just class compliant.

Roland's XV-2020 synthesizer also can do that with each a MIDI in/out.
